tests/test_ec2.py

```diff
@@ -1451,3 +1451,18 @@ def test_regex_filter(self):
         )
         resources = policy.run()
         self.assertGreater(len(resources), 0)
+
+
+class LaunchTemplateTest(BaseTest):
+
+    def test_launch_template(self):
+        session_factory = self.replay_flight_data("test_launch_template")
+        policy = self.load_policy(
+            {
+                "name": "launch-template-test",
+                "resource": "launch-template"
+            },
+            session_factory=session_factory,
+        )
+        resources = policy.run()
+        self.assertEqual(len(resources), 1)
```

Comment:
its good to verify an attribute, like at least name, to assert that we're actually getting back the thing we're expecting. i know we have some tests that historically have done these simple count checks, but most newer tests do some form of attribute verification.

Reply:
added a check on the name, however describe launch templates doesn't return very much information so its a fairly basic check